time.c

```diff
@@ -5591,217 +5591,7 @@ rb_time_zone_abbreviation(VALUE zone, VALUE time)
  *  nanosecond), Time works slower than when integer is used.
  */
 
-/*
```

Comment:
I don't think this change is related to this pull request.

Reply:
I had some sort of conflict with these last two, that I apparently resolved incorrectly.  Solution is to add <tt> /*</tt> (space, slash, asterisk)?

Reply:
Restored.